src/hooks/useStore.js

```diff
@@ -178,25 +178,27 @@ const useStore = create(
 			},
 		],
 		dropdown: [{ isClicked: false }],
+		addFeatureState: false,
+		//appState Functions
 		appState: 'featurelist',
-		stateToFeatureView: () => {
+		stateToFeatureList: () => {
 			set(
 				produce(draft => {
-					draft.appState = 'featureview';
+					draft.appState = 'featurelist';
 				})
 			);
 		},
-		stateToFeatureList: () => {
+		stateToEditFeature: () => {
 			set(
 				produce(draft => {
-					draft.appState = 'featurelist';
+					draft.appState = 'editfeature';
 				})
 			);
 		},
-		stateToEditFeature: () => {
+		stateToProjects: () => {
 			set(
 				produce(draft => {
-					draft.appState = 'editfeature';
+					draft.appState = 'projects';
```

Comment:
instead of writing a lot of functions, there should be one function called "setAppState" this function receives the string you want your appState to be.

Reply:
How does this work? I think, I need a function to declare a state and than query it right on the component (appState = "editfeature" && <EditFeature />)

Reply:
```
appState: null,
setAppState: (appState) => {
			set({ appState });
		},
 ```

Reply:
I dont get it. I think (maybe) you want me to write

setAppState: (appState) => {
			set({ appState = "editfeature" });
		},
		{ 
		       set({ appState = "todolist" });
		}, and so on       
		
In this case, how do I adress a single function in my component? On my version, I set stateToTodolist() and the appState switched to that specific state. Now, I dont have a name beside setAppState, which carries all the setFunctions.

Reply:
you will call the function like this:

```
<button onClick={() => {
        setAppState("running");
    }}>
click!
</button>
```